compiler/rustc_codegen_ssa/src/back/linker.rs

```diff
@@ -1373,11 +1364,6 @@ impl<'a> Linker for PtxLinker<'a> {
         self.cmd.arg("-o").arg(path);
     }
 
-    fn finalize(&mut self) {
```

Comment:
Is there context for this removal that is necessary to understand why this argument is no longer necessary?

Reply:
The necessary part is replaced with `fn reset_per_library_state`, other uses are just random order-independent flags which are moved to `fn add_order_independent_options`.